packages/core/src/components/time-input/test/time-input.ct.ts

```diff
@@ -152,4 +153,25 @@ regressionTest.describe('time input tests', () => {
       );
     }
   );
+
+  regressionTest(
+    'updating component value attribute updates validity',
+    async ({ page }) => {
+      await page.evaluate(() => {
+        document
+          .getElementById('time-input')
+          ?.setAttribute('value', 'invalid-time');
+      });
```

Comment:
There is a better solution using locator:

```suggestion
      const timeInput = page.locator('ix-time-input');
      await timeInput.evaluateHandle((el) => {
       el.setAttribute('value', 'invalid-time');
      });
```

Maybe check other cases also and adapt please.